지미_김지명/2주차_미션/mission1/src/components/TodoForm.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState, type FormEvent } from "react";
+import { useTodo } from "../context/TodoContext";
+
+const TodoForm = () => {
+    const { addTodo } = useTodo();
+
+    const [input, setInput] = useState<string>('');
+    const handleSubmit = (e: FormEvent<HTMLFormElement>) : void => {
+        e.preventDefault();
+        const text = input.trim();
+
+        if (text) {
+            // addTodo
+            addTodo(text);
+            setInput('');
+        }
+    };
+
+  return (
+    <form onSubmit={handleSubmit}
+    className="todo-container__form">
+        <input 
+            type="text"
+            className="todo-container__input"
+            placeholder="할 일 입력"
+            required
+            value={input}
+            onChange={(e): any => setInput(e.target.value)}
```

Comment:
any타입은 최대한 사용하지 않는 것이 좋습니다!

onChange를 넘겨줄 때 저렇게 작성해줘도 좋지만


```suggestion
    const handleChangeInput = (e: React.ChangeEvent<HTMLInputElement>) => {
        setInput(e.target.value);
    };
```
를 작성하고, onChange={handleChangeInput} 과 같이 넘겨주면 더 좋을 것 같습니다!